ci-operator/templates/openshift/installer/cluster-launch-installer-upi-src.yaml

```diff
@@ -0,0 +1,1303 @@
+          platform:
+            aws:
+              zones:
+              - ${AWS_REGION}a
+              - ${AWS_REGION}b
+              - ${AWS_REGION}c
+        compute:
+        - name: worker
+          replicas: 0
+          platform:
+            aws:
+              zones:
+              - ${AWS_REGION}a
+              - ${AWS_REGION}b
+              - ${AWS_REGION}c
+        networking:
+          machineCIDR: ${MACHINE_CIDR}
+        platform:
+          aws:
+            region: ${AWS_REGION}
+            userTags:
+              expirationDate: ${EXPIRATION_DATE}
+        pullSecret: >
+          ${PULL_SECRET}
+        sshKey: |
+          ${SSH_PUB_KEY}
+        EOF
+
+          openshift-install --dir=/tmp/artifacts/installer create manifests
+          rm -f /tmp/artifacts/installer/openshift/99_openshift-cluster-api_master-machines-*.yaml /tmp/artifacts/installer/openshift/99_openshift-cluster-api_worker-machinesets-*.yaml
+        elif [[ "${CLUSTER_TYPE}" == "gcp" ]]; then
+          base_domain="${BASE_DOMAIN:-origin-ci-int-gce.dev.openshift.com}"
+          cat > /tmp/artifacts/installer/install-config.yaml << EOF
+        apiVersion: v1
+        baseDomain: ${base_domain}
+        metadata:
+          name: ${CLUSTER_NAME}
+        controlPlane:
+          name: master
+          replicas: 3
+        compute:
+        - name: worker
+          replicas: 0
+        platform:
+          gcp:
+            projectID: ${GCP_PROJECT}
+            region: ${GCP_REGION}
+        pullSecret: >
+          ${PULL_SECRET}
+        sshKey: |
+          ${SSH_PUB_KEY}
+        EOF
+
+          openshift-install --dir=/tmp/artifacts/installer create manifests
+          rm -f /tmp/artifacts/installer/openshift/99_openshift-cluster-api_master-machines-*.yaml
+          rm -f /tmp/artifacts/installer/openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+          sed -i "s;mastersSchedulable: true;mastersSchedulable: false;g" /tmp/artifacts/installer/manifests/cluster-scheduler-02-config.yml
+        elif [[ "${CLUSTER_TYPE}" == "vsphere" ]]; then
+            base_domain="${BASE_DOMAIN:-origin-ci-int-aws.dev.rhcloud.com}"
+            # Get user and password from TFVARS_PATH
+            export VSPHERE_USER=$(grep -oP 'vsphere_user="\K[^"]+' ${TFVARS_PATH})
+            export VSPHERE_PASSWORD=$(grep -oP 'vsphere_password="\K[^"]+' ${TFVARS_PATH})
+            export OVA_URL="$(jq -r '.baseURI + .images["vmware"].path' /var/lib/openshift-install/rhcos.json)"
+            export VM_TEMPLATE="${OVA_URL##*/}"
+            mkdir /tmp/rhcos
+
+            cat > /tmp/rhcos/rhcos.json << EOF
+        {
+           "DiskProvisioning": "thin",
+           "MarkAsTemplate": false,
+           "PowerOn": false,
+           "InjectOvfEnv": false,
+           "WaitForIP": false,
+           "Name": "${VM_TEMPLATE}"
+        }
+        EOF
+            cat > /tmp/artifacts/installer/install-config.yaml << EOF
+        apiVersion: v1beta4
```

Comment:
@markusthoemmes apiVersion should be v1.  I need to change that in the other template as well.